test/prod.spec.js

```diff
@@ -13,7 +13,7 @@ describe('express errors handler in prod', function () {
 	let errorsHandler;
 	const readTimeoutError = new fetchres.ReadTimeoutError();
 	const badServerError = new fetchres.BadServerResponseError(418);
-	const error = new Error('potato');
+	let errorToPassThrough = new Error('potato');
```

Comment:
can still be a `const` as we only ever modify its properties, not the object itself